### PR TITLE
Video lib initial frontend done

### PR DESCRIPTION
**Pull request by theshivamverma**
[App link](https://indie-songs.netlify.app/)

### CODE REVIEW

File: README.old.md [hunk 1]
```diff
@@ -0,0 +1 @@
+### This repo is for video library app
```
Comment (h-harsh) [inline README.old.md]: More description to readme file can be given

File: src/App.js [hunk 2]
```diff
@@ -1,8 +1,25 @@
+import { useState } from "react";
+import { Routes, Route } from "react-router-dom"
+import LeftNavbar from "./components/LeftNavbar/LeftNavbar";
+import { Playlist } from "./components/playlist";
+import PlaylistDetail from "./components/playlist/PlaylistDetail";
+import Topnavbar from "./components/Topnavbar/Topnavbar";
+import VideoList from "./components/VideoList";
+import VideoPage from "./components/VideoPage";
```
Comment (h-harsh) [inline src/App.js]: Make a better file structure and use index file for importing them all at once

File: src/components/Topnavbar/Topnavbar.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+export default function Topnavbar({ setLeftMenu }) {
+  return (
+    <nav class="nav top-fixed p-1 box-shadow-down">
+      <h1 class="font-size-m">Indie Songs</h1>
+      <button class="btn btn-icon" onClick={() => setLeftMenu(true) }>
+        <i class="fas fa-bars icon-med"></i>
```
Comment (h-harsh) [inline src/components/Topnavbar/Topnavbar.js]: className has to be used instead of class

File: src/components/VideoCard.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+          alt=""
+          src={`https://img.youtube.com/vi/${videoId}/hqdefault.jpg`}
+        />
+      </Link>
+      <div className="flex justify-sb align-center mt-1">
+        <Link to={`/video/${videoId}`} state={{ title }}>
+          <h3 className="video-title medium font-size-sm">{title}</h3>
+        </Link>
+        <button
+          class="btn btn-icon ml-1"
```
Comment (h-harsh) [inline src/components/VideoCard.js]: use className instead of class

File: src/components/VideoList.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+
+// const videoIds = [
+//   "0P3Gt-60yLc",
+//   "AVM4J0qvdKw",
+//   "5eIhSCFnvPc",
+//   "PJWemSzExXs",
+//   "7YbfDiCC-qk",
+//   "jJPMnTXl63E",
+// ];
```
Comment (h-harsh) [inline src/components/VideoList.js]: remove unused code

File: src/components/playlist/Playlist.js [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+                            playlistDispatch({
+                              type: "REMOVE_PLAYLIST",
+                              payload: { playlistName: playlistItem.name },
+                            })
+                          }
+                        >
+                          <icon className="fas fa-trash icon-med colorAlertRed"></icon>
+                        </button>
+                      </div>
+                      {playlistItem.videos.map((video) => {
```
Comment (h-harsh) [inline src/components/playlist/Playlist.js]: destructure element in map

**Review by h-harsh — CHANGES_REQUESTED** [review]

**Comment by Akshay-Laxmipathi** [comment]
Try to implement a feature that prevents users from creating a new playlist with an existing playlist name.

**Pull request merged**